lib/KeyboardAwareHOC.js

```diff
@@ -298,18 +298,17 @@ function KeyboardAwareHOC(
       if (keyboardOpeningTime === undefined) {
         keyboardOpeningTime = this.props.keyboardOpeningTime || 0
       }
-      setTimeout(() => {
-        if (!this.mountedComponent) {
-          return
-        }
-        const responder = this.getScrollResponder()
-        responder &&
-          responder.scrollResponderScrollNativeHandleToKeyboard(
-            reactNode,
-            extraHeight,
-            true
-          )
-      }, keyboardOpeningTime)
+      if (this.mountedComponent) {
+        UIManager.measureLayout(
```

Comment:
@taichi-jp hey, why do you use this approach? I have removed `setTimeout` only and this works fine.
Also, with your approach, auto-scroll doesn't work when I press on a first input in the scrollview 🤷‍♂️

Reply:
I still had issues after only removing the setTimeout. Scrolling works for me on first input as well (works on my machine... ;)).

Reply:
i see :) hm, will try to find a reason 🕵️‍♂️

Reply:
Implementing the given solution works fine for me. When should we able to merge or find a solution to this issue? Thx

Reply:
When solution will be merged? Thanks

Reply:
I hope it gets merged in 2024